@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -307,19 +326,20 @@ export class TextElementsRenderer {
 
     /**
      * Places text elements for the current frame.
-     * @param tileTextElementsChanged Indicates whether there's been any change in the text elements
-     * to place since the last call to this method (last frame).
+     * @param dataSourceTileList List of tiles to be rendered for each data source.
+     * @param projection The view's projection.
+     * @param time Current frame time.
      * @param tilesChanged Indicates whether there's been any change in the tiles rendered since the
      * last call to this method (last frame).
-     * @param time Current frame time.
-     * @param frameNumber Current frame number.
      */
     placeText(
-        tileTextElementsChanged: boolean,
-        tilesChanged: boolean,
+        dataSourceTileList: DataSourceTileList[],
+        projection: Projection,
```

Comment:
ditto, could this be part of the ViewState?

Reply:
Similar reason. I didn't want to give all methods in TextElementsRenderer access to the projection, it's only needed for one thing (to compute the world offset for the tiles).

Reply:
good point